server/src/main/java/com/paypal/selion/utils/ConfigParser.java

```diff
@@ -48,6 +48,10 @@
      *         represented by the JSON file passed via the JVM argument <b>SeLionConfig</b>
      */
     public static ConfigParser parse() {
+        LOGGER.entering();
+        if (parser == null) {
```

Comment:
@mach6 Yeah I have reverted the previous commit. Issue is SeLion not able to read the SeLionConfig.json file from selionHome directory specified from java arguments because of previous change. Since we are creating static instance for ConfigParser, always it will load the SeLionConfig.json from jar file, not the specified file via setConfigFile(String file). Here ConfigParser getting initialized before we are setting config file. So we need lazy initialization to make it work,